### PR TITLE
Add eomt instance segmentation docs

### PR DESCRIPTION
**Pull request by guarin**
## What has changed and why?

* Add docs
* Add colab
* Add `threshold` arg to `predict`

[Instance Segmentation - LightlyTrain documentation.pdf](https://github.com/user-attachments/files/23482938/Instance.Segmentation.-.LightlyTrain.documentation.pdf)
[lightly_train - LightlyTrain documentation - 1.pdf](https://github.com/user-attachments/files/23482940/lightly_train.-.LightlyTrain.documentation.-.1.pdf)

## How has it been tested?

* Manually

Will test again once checkpoints are ready and uploaded

## Did you update [CHANGELOG.md](../CHANGELOG.md)?

- [ ] Yes
- [x] Not needed (internal change)

## Did you update the documentation?

- [ ] Yes
- [x] Not needed (internal change without effects for user)

### CODE REVIEW

File: docs/source/instance_segmentation.md [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+    )
+```
+
+During training, the best and last model weights are exported to
+`out/my_experiment/exported_models/`, unless disabled in `save_checkpoint_args`:
+
+- best (highest validation mask mAP): `exported_best.pt`
+- last: `exported_last.pt`
+
+You can use these weights to continue fine-tuning on another task by loading the
```
Comment (stegmuel) [inline docs/source/instance_segmentation.md]: You can use these weights to continue fine-tuning on another dataset?

File: docs/source/instance_segmentation.md [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+
+### DINOv3 Models
+
+- `dinov3/vits16-eomt`
+- `dinov3/vits16plus-eomt`
+- `dinov3/vitb16-eomt`
+- `dinov3/vitl16-eomt`
+- `dinov3/vitl16plus-eomt`
+- `dinov3/vith16plus-eomt`
+- `dinov3/vit7b16-eomt`
```
Comment (stegmuel) [inline docs/source/instance_segmentation.md:271]: The non-fine-tuned models don't need the "inst" suffix?

File: docs/source/instance_segmentation.md [hunk 3]
```diff
@@ -0,0 +1,420 @@
...
+There are two distinct ways to continue training, depending on your intention.
+
+### Resume Interrupted Training
+
+Use `resume_interrupted=True` to **resume a previously interrupted or crashed training run**.
+This will pick up exactly where the training left off.
+
+- You **must use the same `out` directory** as the original run.
+- You **must not change any training parameters** (e.g., learning rate, batch size, data, etc.).
+- This is intended for continuing the *same* run without modification.
```
Comment (stegmuel) [inline docs/source/instance_segmentation.md]: **same** ?

**Review by stegmuel — COMMENTED** [review]
Left minor comments/questions.

File: docs/source/instance_segmentation.md (hunk 2 above)
Comment (guarin) [reply docs/source/instance_segmentation.md:271]: No, when training you call `train_instance_segmentation`, therefore the task is known. But the pretrained weights need the `inst` to disambiguate between pretrained semantic/instance/panoptic segmentation weights.
